plugins/node/opentelemetry-instrumentation-redis/src/internal-types.ts

```diff
@@ -17,6 +17,7 @@ export interface RedisPluginClientTypes {
   options?: {
     host: string;
     port: string;
+    db: string;
```

Comment:
According to `@types/redis`, this option can also be `string` which we need to convert to `number` when recording into an attribute:

```js
    db?: string | number | undefined;
```